dms/models/dms_file.py

```diff
@@ -172,6 +184,8 @@ def _update_content_vals(self, vals, binary):
         )
         if self.storage_id.save_type == "file":
             new_vals["content_file"] = self.content
+        elif self.storage_id.save_type == "attachment":
+            new_vals["attachment_id"] = self.content
```

Comment:
It's very strange because in V12 don't work https://github.com/OCA/dms/pull/24#issuecomment-703899342.
Solved (V12) with this changes:
```suggestion
                if self.storage_id.save_type in ["file", "attachment"]:
            new_vals["content_file"] = self.content
```